structure/bitmap.go

```diff
@@ -0,0 +1,275 @@
+package structure
+
+import (
+	"bytes"
+	"errors"
+	"github.com/ByteStorage/FlyDB/config"
+	"github.com/ByteStorage/FlyDB/engine"
+	_const "github.com/ByteStorage/FlyDB/lib/const"
+	"github.com/bits-and-blooms/bitset"
+)
+
+type BitmapStructure struct {
+	db *engine.DB
+}
+type BitSet struct {
+	b *bitset.BitSet
+}
+type BitOperation string
+
+const (
+	BitAndOperation BitOperation = "AND"
+	BitOrOperation  BitOperation = "OR"
+	BitXorOperation BitOperation = "XOR"
+	BitNotOperation BitOperation = "NOT"
+)
+
+func NewBitmap(options config.Options) (*BitmapStructure, error) {
```

Comment:
Can you write detailed annatations for your code?  For example, write the function's purpose and how to use it before the function.  This may add to your workload, but when someone else is learning our source code, he may seem confused without annatations .

Thank you for your understanding.

Reply:
@qishenonly I've made the suggested changes; I hope it helps.

Reply:
This is great, thank you!